sklearn/ensemble/forest.py

```diff
@@ -79,6 +79,56 @@ def _generate_sample_indices(random_state, n_samples):
     return sample_indices
 
 
+def _get_class_distribution(y):
+    """Private function used to fit function.
```

Comment:
No need to mention this is private. It is more useful to mention what the function is doing:

```suggestion
    """Compute the class distributions and associated statistics.
```